### PR TITLE
GH-860 Table Show-Hide Header

### PR DESCRIPTION
**Pull request by dtsnell4**
Adds a publish parameter that will show or hide the Table Widget's header if true or false.

Fixes GH-860

Signed-off-by: Dan Snell <Dan.Snell@lexisnexis.com>

### CODE REVIEW

**Comment by dtsnell4** [comment]
@GordonSmith @jbrundage @mlzummo Please review

File: src/other/Table.js [hunk 1]
```diff
@@ -548,8 +558,11 @@
                 ;
             } else {
                 rowWrapperWidth = 0; 
+                tableMarginHeight = 0;
             }
-            newTableHeight = parseInt(context.headerDiv.node().style.height) - parseInt(colWrapperHeight);
+                            console.log(tableMarginHeight);
```
Comment (dtsnell4) [inline src/other/Table.js]: remove console.log

File: src/other/Table.js [hunk 2]
```diff
@@ -570,14 +583,14 @@
             ;
             rowsWrapper
                 .style("width", rowWrapperWidth + "px")
-                .style("height", newTableHeight + "px")
+//                .style("height", newTableHeight + "px")
```
Comment (GordonSmith) [inline src/other/Table.js]: Remove commented code

**Comment by GordonSmith** [comment]
Seems to work well:  http://rawgit.com/dtsnell4/Visualization/GH-860_Hide_Table_Header/demos/dermatology.html?src/other/Table

**Comment by dtsnell4** [comment]
@GordonSmith comments and console log removed.

**Pull request merged**